lib/guardian/user_guardian.rb

```diff
@@ -167,4 +167,7 @@ def can_upload_user_card_background?(user)
     (is_me?(user) && user.has_trust_level?(SiteSetting.min_trust_level_to_allow_user_card_background.to_i)) || is_staff?
   end
 
+  def can_delete_sso_record?(user)
+    SiteSetting.enable_sso && user && is_staff?
```

Comment:
Should this be `is_admin?` instead? `is_staff?` will return true for moderators and admins.